<!DOCTYPE html>
<html>
<head>
<title>mzTab File Format | Crux Template</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="stylesheet" type="text/css" href="../styles.css">
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-26136956-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
<script type="text/javascript">
	// Main Menu
	$( document ).ready(function() {
			var pull 		= $('.btn');
				menu 		= $('nav ul');
				menuHeight	= menu.height();

			$(pull).on('click', function(e) {
				e.preventDefault();
				menu.slideToggle();
			});

			$(window).resize(function(){
	    		var w = $(window).width();
	    		if(w > 320 && menu.is(':hidden')) {
	    			menu.removeAttr('style');
	    		} 
			});
	});
</script>
</head>
<body>
	<div class="page-wrap">
	 	<nav>
			<div class="btn">
		    </div>
		    <img src="../images/crux-logo.png" id="logo"></a>
			<ul id="navitems">
			    <li><a href="../index.html">Home</a></li>
			    <li><a href="../download.html">Download</a></li>
			    <li><a href="../fileformats.html">File Formats</a></li>
		        <li><a href="http://groups.google.com/group/crux-users">Contact</a></li> <!--Link to google support board-->
		    </ul>
		</nav>
		<div id="content">
		<!-- START CONTENT -->
			<h2>mzTab  file format</h2>

			<p>
				The mzTab data exchange format is used for communicating mass-spectrometry-based proteomics and metabolomics experimental results. 
				<cite>
					Griss J et al.
					The mzTab data exchange format: communicating mass-spectrometry-based proteomics and metabolomics experimental results to a wider audience. 
					Mol Cell Proteomics. 2014 Oct;13(10):2765-75. doi: 10.1074/mcp.O113.036681. Epub 2014 Jun 30. PMID: 24980485; PMCID: PMC4189001.
				</cite>
				<br>
				<br>
				The mzTab format is a tab-delimited text file consisting of a metadata section, a protein section, 
				a peptide section which is not recommended in "Identification" files, and a PSM section as well as a small molecule section.
				Crux supports the mzTab format for both "Quantification" and "Identification" files.
				Crux supports the version 1 of the mzTab format.
				<br>
				For a detailed explanation of the mzTab format, please see the <a href="https://github.com/HUPO-PSI/mzTab/blob/master/specification_document-releases/1_0-Proteomics-Release/mzTab_format_specification.pdf">mzTab format specification</a>
			</p>
			<center>
				<p>Meta Data Section</p>
				<table border="1" id="tabformat">
					<!-- table headers-->
					<tr>
						<th>Name</th>
						<th>Description</th>
					</tr>
					<tr>
						<td>mzTab-version</td>
						<td>It specificaties the mzTab version being used.</td>
					</tr>
					<tr>
						<td>mzTab-mode</td>
						<td>
							It specifies the mode of the experiment which was conducted 
							(either "Summary" or "Complete").
						</td>
					</td>
					<tr>
						<td>mzTab-type</td>
						<td>
							It specifies the type of experiment which was conducted 
							(either "Quantification" or "Identification").
						</td>
					</tr>
					<tr>
						<td>description</td>
						<td>
							It contains a short description of the experiment that was conducted.
						</td>
					</td>
				</table> 
				<p>Protein Section</p>
				<table border="1" id="tabformat">
					<!-- table headers-->
					<tr>
						<th>Name</th>
						<th>Description</th>
					</tr>
					<tr>
						<td>file</td>	
						<td>The name of the file containing the scan.</td>
						</tr>
						<tr>
						<td>scan</td>	
						<td>The identifying number of each scan.</td>
						</tr>
						<tr>
						<td>charge</td>	
						<td>The charge state for this spectrum in this PSM.</td>
						</tr>
						<tr>
						<td>spectrum precursor m/z</td>	
						<td>The observed m/z of the spectrum precursor ion.</td>
						</tr>
						<tr>
						<td>spectrum neutral mass</td>
						<td>The computed mass of the spectrum precursor at the given charge
							state. This is equal to the precursor m/z minus the mass of a proton
							(1.00727646677 Da), all multiplied by the charge.</td>
						</tr>
						<tr>
						<td>peptide mass</td>	
						<td>The mass of the peptide sequence, computed as the sum of the
							amino acid masses plus the mass of water (18.010564684 Da or
							18.0153 Da, depending on whether we are using monoisotopic or
							average mass).</td>
						</tr>
						<tr>
						<td>delta_cn</td>
						<td>The normalized difference in XCorr for this PSM
							relative to the next ranked PSM for the same spectrum
							and charge. The denominator for normalization is the
							maximum of the current XCorr and 1.0.
							If <code>exact-p-value=T</code>, then the difference
							is computed between -log(p-value) rather than XCorr,
							and no normalization is applied.</td>	
						</tr>
						<tr>
						<td>delta_lcn</td>
						<td>Similar to delta_cn, except that the difference
							is computed with respect to the lowest reported XCorr
							score for a given spectrum and charge state.</td>	
						</tr>
						<tr>
						<td>xcorr score	</td>
						<td>The SEQUEST-type cross correlation score.
							This column only appears when <code>exact-p-value=F</code>.</td>
						</tr>
						<tr>
						<td>tailor score</td>
						<td></td>
						</tr>
						<tr>	
						<td>xcorr rank</td>	
						<td>The rank of this PSM when sorted by xcorr.</td>
						</tr>
						<tr>
						<td>distinct matches/spectrum</td>	
						<td>The number of unique candidate peptides in the database found for this
							spectrum.  Note that this is always the number of target candidate
							peptides, even if the PSM involves a decoy peptide.</td>
						</tr>
						<tr>
						<td>sequence</td>	
						<td>The peptide sequence, not including any modifications.</td>
						</tr>
						<tr>
						<td>modifications</td>
						<td>Variable modifications applied to the sequence.</td>	
						</tr>
						<tr>
						<td>cleavage type</td>	
						<td>The cleavage rules for generating this peptide based on the
							user-specified enzyme specificity.</td>
						</tr>
						<tr>
						<td>protein id</td>	
						<td>A comma-separated list of proteins in which this peptide
							appears. Optionally, the protein name may be followed by a number
							in parentheses giving the start position of the peptide in the
							protein.</td>
						</tr>
						</tr>
						<td>flanking aa	</td>
						<td>The amino acids that precede and follow this
							peptide in the parent protein ID.  If the peptide
							occurs in more than one protein, then this column
							will contain a comma-separated list of pairs of amino
							acids.</td>
						</tr>
						<tr>
						<td>target/decoy</td>	
						<td></td>
						</tr>
						<tr>
						<td>original target sequence</td>
						<td>The unmodified target sequence. For a target PSM,
							the value in this column will be identical to the
							value in the "sequence" column.  For a decoy PSM,
							this column will contain the corresponding target
							sequence.</td>
						</tr>

				</table> 
			</center>
		<!-- END CONTENT -->
		</div>
	</div>

<footer class="site-footer">
	<div id="centerfooter">
		<div class="footerimportantlinks">
			<img src="../images/linkicon.png" style="width:16px; height:16px"><h3>Important links</h3>
			<ul>
				<li><a href="../faq.html">Crux FAQ</a></li>
				<li><a href="../glossary.html">Glossary of terminology</a></li>
				<li><a href="http://scholar.google.com/citations?hl=en&user=Rw9S1HIAAAAJ">Google Scholar profile</a></li>
				<li><a href="https://github.com/crux-toolkit/crux-toolkit">GitHub</a></li>
				<li><a href="../release-notes.html">Release Notes</a></li>
				<li><a href="https://mailman1.u.washington.edu/mailman/listinfo/crux-users" title="Receive announcements of new versions">Join the mailing list</a></li>
				<li><a href="http://www.apache.org/licenses/LICENSE-2.0">Apache license</a></li>
				<li><a href="http://groups.google.com/group/crux-users">Support Board</a></li>
			</ul>
		</div>
		<div class="footerimportantlinks tutoriallinks">
			<img src="../images/tutorialicon.png" style="height:16px"><h3>Tutorials</h3>
			<ul>
				<li><a href="../tutorials/install.html">Installation</a></li>
				<li><a href="../tutorials/gettingstarted.html">Getting started with Crux</a></li>
				<li><a href="../tutorials/search.html">Running a simple search using Tide and Percolator</a></li>
				<li><a href="../tutorials/customizedsearch.html">Customization and search options</a></li>
				<li><a href="../tutorials/spectralcounts.html">Using spectral-counts</a></li>
			</ul>
		</div>
		<div id="footertext">
			<p>
				The original version of Crux was written by Chris Park and Aaron Klammer
				under the supervision
				of <a href="http://www.gs.washington.edu/faculty/maccoss.htm">Prof. Michael
				MacCoss</a>
				and <a href="http://noble.gs.washington.edu/~noble">Prof. William
				Stafford Noble</a> in the Department of Genome Sciences at the
				University of Washington, Seattle.  Website by <a href="http://www.yuvalboss.com/">Yuval Boss</a>. 
				<br />The complete list of contributors
				can be found <a href="../contributors.html">here</a>.
				<br />
				<br />
				Maintenance and development of Crux is funded by the <a href="https://www.nih.gov/">National Institutes of Health</a> award P41 GM103533. 
			</p>
		</div>
	</div>
</footer>
	
</body>

</html>
